staging/src/k8s.io/kube-aggregator/pkg/apiserver/handler_proxy_test.go

```diff
@@ -774,6 +779,106 @@ func TestGetContextForNewRequest(t *testing.T) {
 
 }
 
+func TestTracerProvider(t *testing.T) {
+	fakeRecorder := tracetest.NewSpanRecorder()
+	otelTracer := sdktrace.NewTracerProvider(sdktrace.WithSpanProcessor(fakeRecorder))
+	t.Run("sets correct traceparent header", func(t *testing.T) {
```

Comment:
why a subtest if there is only one? `t.Run` is rather used for table tests.

Reply:
👍 removed